ReleaseNotes.md

```diff
@@ -17,7 +17,7 @@ You can create your own custom build of Infusion using the [grunt build script](
 ## Demos ##
 
 Infusion ships with a demos for seeing all of the components in action. You can
-find them in the _**demos**_ folder in the release bundle or on our [web site](http://fluidproject.org/products/infusion/infusion-demos/).
+find them in the _**demos**_ folder in the release bundle or on our [build site](http://build.fluidproject.org/infusion/demos/).
```

Comment:
Well, the surrounding text talks specifically about demos. The build site is nicer looking, but it also includes all the tests... Do you think the nice listing of demos is worth the extra clutter and risk of confusion?